### PR TITLE
added odds data automatically

### PR DESCRIPTION
**Pull request by ayoub377**
facilitate input of odds. odds scraped and integrated in the main method.

### CODE REVIEW

**Comment by kyleskom** [comment]
Will be testing this shortly.

**Comment by kyleskom** [comment]
What additional libraries are needed for this? We need to add them the the requirements.txt

File: src/Utils/tools.py [hunk 1]
```diff
@@ -48,5 +49,33 @@ def create_todays_games(input_list):
...
+            if i % 2 == 0:
+                team_name = match_info[0].get_text()
+                team_money_line_odds = match_info[1].get_text()
+                dict_res[team_name] = team_money_line_odds
+                i += 1
+            elif i % 2 != 0:
+                team_name = match_info[1].get_text()
+                team_money_line_odds = match_info[0].get_text()
+                dict_res[team_name] = team_money_line_odds
+                i += 1
```
Comment (kyleskom) [inline src/Utils/tools.py:79]: This works but would be great to do the under / over as well.

File: main.py [hunk 2]
```diff
@@ -25,10 +28,14 @@ def createTodaysGames(games, df):
     for game in games:
         home_team = game[0]
         away_team = game[1]
+        home_abrev = game[2]
+        away_abrev = game[3]
         todays_games_uo.append(input(home_team + ' vs ' + away_team + ': '))
```
Comment (kyleskom) [inline main.py:33]: Remove and grab under / over (totals) as well.

File: main.py [hunk 3]
```diff
@@ -1,6 +1,9 @@
 import argparse
 import pandas as pd
 import tensorflow as tf
+
+#definition of method
+from src.Utils.tools import get_odds
```
Comment (kyleskom) [inline main.py:6]: This import can be combined with line 
 `from src.Utils.tools import get_json_data, to_data_frame, get_todays_games_json, create_todays_games`

File: main.py [hunk 4]
```diff
@@ -49,8 +57,9 @@ def main():
     data = get_todays_games_json(todays_games_url)
     games = create_todays_games(data)
     data = get_json_data(data_url)
+    odds = get_odds()
     df = to_data_frame(data)
-    data, todays_games_uo, frame_ml, home_team_odds, away_team_odds = createTodaysGames(games, df)
+    data, todays_games_uo, frame_ml, home_team_odds, away_team_odds = createTodaysGames(games, odds, df)
```
Comment (kyleskom) [inline main.py:62]: Try to run this with `python main.py -A`, I think the xgboost model path breaks with changes. Looks like a minor fix.

**Review by kyleskom — CHANGES_REQUESTED** [review]

**Comment by ayoub377** [comment]
Beautifulsoup

File: main.py (hunk 2 above)
Comment (ayoub377) [reply main.py:33]: i think over/under need to be optional for user because each user will have a specific over/under total

Comment (kyleskom) [reply main.py:33]: I think this would be the same case for team lines as well. Odds will be slightly different between books.

**Comment by kyleskom** [comment]
Im still interested in this if you are still working on it!

**Comment by ayoub377** [comment]
I am working on some final exams as soon as possible I will try to finish
it its still on my plans.

**Comment by kyleskom** [comment]
No worries at all no rush! Was just looking at the open pr's today and thought to follow up. Best of luck on your exams!

**Comment by kyleskom** [comment]
Done with #53

**Pull request closed without merging**